package.json

```diff
@@ -47,6 +47,10 @@
     "create-react-class": "15.5.3",
     "creditcards": "2.1.2",
     "cross-env": "4.0.0",
+    "d3-array": "1.2.0",
```

Comment:
just to make sure, what version of `npm` did you use when regenerating the shrinkwrap? There are some order changes in there that make me suspect you're not on `3.10.10`.